lib/src/screens/receive/receive_page.dart

```diff
@@ -165,7 +165,7 @@ class ReceiveBodyState extends State<ReceiveBody> {
                                             decimal: true),
                                     inputFormatters: [
                                       FilteringTextInputFormatter.deny(
-                                          RegExp('[\\-|\\ |\\,]'))
+                                          RegExp('[\\-|\\ ]'))
```

Comment:
This regex doesn't look right: `[...]` is a character class, not a group, so this is going to match any of: `\`, `-`, `|`, ` `.  But it seems the intention was to match any of `-`, ` `, or `,` (originally), which would be a regex of just `'[- ]'`